src/app/copy/block-copy/block-copy.component.less

```diff
@@ -0,0 +1,50 @@
+.pfng-block-copy {
+  &-inner-container {
+    border: 1px solid #bbb;
+  }
+  &-preview {
+    display: flex;
+    align-items: stretch;
```

Comment:
`align-items: stretch;` is the default value for `align-items` so that can be removed.